tests/unit/synapseclient/unit_test_commandline.py

```diff
@@ -2,8 +2,8 @@
 
 """
 
-from nose.tools import assert_equals, assert_true, assert_false
-from mock import patch
+from nose.tools import assert_equal, assert_true, assert_false
+from mock import patch, Mock
```

Comment:
Small thing, but mock is now part of `unittest`. Eventually the backport will probably go away, so all imports will most likely be 

```
from unittest.mock....
```

Reply:
Yes this is true, good point. The current project uses the backport throughout so I will make a separate pull request to search and replace all uses of the backport (after which we can remove it as a test_requires dependency).